README.md

```diff
@@ -10,6 +10,12 @@ increase input performance.
 # Install
 * Arch Linux: install from AUR: [Stable release](https://aur.archlinux.org/packages/chewing-editor/), [Development](https://aur.archlinux.org/packages/chewing-editor-git/)
 * Mageia: install via `urpmi chewing-editor` - available in cauldron, Mageia 6 and later.
+* Ubuntu Linux:
+  * install from Ubuntu [APT repository](http://packages.ubuntu.com/search?keywords=chewing-editor) since Ubuntu 16.04 LTS (Xenial Xerus)
+    * `# apt-get update chewing-editor`
```

Comment:
I saw you updated the PR, is it ready? Pardon me for early review.

Two concerns here:
1. `apt-get update` doesn't take a parameter.
2. `#` is comment keyword in shell script, is it better use a `$` or omit it at all?